in/index.html.mako

```diff
@@ -77,6 +77,10 @@
           data-text-off="Compare Selected">
           Compare Selected
         </button>
+
+        <button class="btn btn-warning btn-clear">
```

Comment:
This button really stands out with the `btn-warning` style, but I don't think it'll get used all that often. What about hiding it if the current selections are the defaults, and/or making it just a normally styled button when shown?